.travis.yml

```diff
@@ -2,7 +2,23 @@ language: java
 # OpenTripPlanner requires Java 8 and Travis doesn't (yet) support OpenJDK 8
 jdk:
```

Comment:
Is this pattern of usage allowed/encouraged by Travis CI? I imagine they might not like us triggering 8 long builds for every single commit. Maybe they're fine with us using the additional resources, but do we have a page stating that's OK?

Also, what is the impact on build artifact upload and deployment? Is this going to cause 8 separate artifacts to be deployed to Maven and S3? What about releases, which build will be the one used in the release? Won't all of them try to release?

Reply:
I know a lot of projects that uses that very pattern and I didn't saw any complain by Travis
We can certainly try to optimize that usage of resource, but I think that we should at least do this full test before pushing a new release.